README.md

```diff
@@ -43,30 +70,50 @@ The Gini Vision Library consists of four main screens:
 
 * Onboading: Provides useful hints to the user on how to take a perfect photo of a document.
 * Camera: The actual camera screen to capture the image of the document.
-* Review: Offers the opportunity to the user to check the sharpness of the image and to rotate it into reading direction, if necessary.
-* Analysis: Provides a UI for the analysis process of the document by showing the user a loading indicator and the image of the document.
+* Review: Offers the opportunity to the user to check the sharpness of the image and to rotate it
```

Comment:
Not sure about this section in the README, there are more component apart from these and as a developer I wouldn't find it very useful. On iOS I decided to remove it from here and added to an specific Integration guide.
What do you think? 
Also ~Onboading~ Onboarding.

Reply:
I will keep it for now. It says these are the _main_ screens. We don't have an integration guide yet covering the whole library. Most information is in the Javadoc and in the update guides.